data/names/ru.json

```diff
@@ -5870,7 +5870,624 @@
+      [ "Стоукс", "Stokes" ],
+      [ "Пейдж", "Page" ],
+      [ "Стрейндж", "Strange" ],
+      [ "Стрэттон", "Stratton" ],
+      [ "Стэнфилд", "Stanfield" ],
+      [ "Сунига", "Zuniga" ],
+      [ "Сэй", "Seay" ],
+      [ "Сэйлор", "Saylor" ],
+      [ "Таварес", "Tavares" ],
+      [ "Таннер", "Tanner" ],
+      [ "Тейшейра", "Teixeira" ],
+      [ "Теллез", "Tellez" ],
+      [ "Тернер", "Turner" ],
+      [ "Титус", "Titus" ],
+      [ "Толберт", "Tolbert" ],
+      [ "Мерчант", "Merchant" ],
+      [ "Траммелл", "Trammell" ],
+      [ "Траутман", "Troutman" ],
+      [ "Труонг", "Truong" ],
+      [ "Трэхан", "Trahan" ],
+      [ "Тэкетт", "Tackett" ],
+      [ "Тэм", "Tam" ],
+      [ "Бэйлс", "Bales" ],
+      [ "Тёрпин", "Turpin" ],
+      [ "Уайлс", "Wiles" ],
+      [ "Уиллер", "Wheeler" ],
+      [ "Уиллингем", "Willingham" ],
+      [ "Уиллоби", "Willoughby" ],
+      [ "Уинн", "Winn" ],
+      [ "Уиттен", "Whitten" ],
+      [ "Уиттингтон", "Whittington" ],
+      [ "Смарт", "Smart" ],
+      [ "Уоркмен", "Workman" ],
+      [ "Уотерман", "Waterman" ],
+      [ "Уэбстер", "Webster" ],
+      [ "Уэллетт", "Ouellette" ],
+      [ "Уэлч", "Welch" ],
+      [ "Фалкон", "Falcon" ],
+      [ "Фаррис", "Farris" ],
+      [ "Фейр", "Fair" ],
+      [ "Фитцпатрик", "Fitzpatrick" ],
+      [ "Флеминг", "Fleming" ],
+      [ "Фой", "Foy" ],
+      [ "Фоли", "Foley" ],
+      [ "Фоллон", "Fallon" ],
+      [ "Фомэн", "Forman" ],
+      [ "Фонтено", "Fontenot" ],
+      [ "Траут", "Trout" ],
+      [ "Франк", "Frank" ],
+      [ "Франкс", "Franks" ],
+      [ "Франс", "France" ],
+      [ "Фрейли", "Fraley" ],
+      [ "Фрейтас", "Freitas" ],
+      [ "Фримен", "Freeman" ],
+      [ "Фуртадо", "Furtado" ],
+      [ "Фэган", "Fagan" ],
+      [ "Хадсон", "Hudson" ],
+      [ "Хайтауер", "Hightower" ],
+      [ "Хакетт", "Hackett" ],
+      [ "Хамель", "Hamel" ],
+      [ "Хаммондс", "Hammonds" ],
+      [ "Харгроув", "Hargrove" ],
+      [ "Хатсон", "Hutson" ],
+      [ "Хау", "Howe" ],
+      [ "Хаятт", "Hyatt" ],
+      [ "Хейден", "Hayden" ],
+      [ "Хейн", "Hein" ],
+      [ "Хейр", "Hare" ],
+      [ "Хеллер", "Heller" ],
+      [ "Хелмс", "Helms" ],
+      [ "Хелтон", "Helton" ],
+      [ "Хенди", "Handy" ],
+      [ "Херли", "Hurley" ],
+      [ "Херрик", "Herrick" ],
+      [ "Херрон", "Herron" ],
+      [ "Хиггинботэм", "Higginbotham" ],
+      [ "Хинкль", "Hinkle" ],
+      [ "Хинонес", "Quinones" ],
+      [ "Коттон", "Cotton" ],
+      [ "Холл", "Hall" ],
+      [ "Хоули", "Hawley" ],
+      [ "Кристиан", "Christian" ],
+      [ "Хэлм", "Helm" ],
+      [ "Хэнкинс", "Hankins" ],
+      [ "Хёрн", "Hearn" ],
+      [ "Флауэрс", "Flowers" ],
+      [ "Чёрч", "Church" ],
+      [ "Чайлдс", "Childs" ],
+      [ "Чайлдресс", "Childress" ],
+      [ "Шампейн", "Champagne" ],
+      [ "Чампион", "Champion" ],
+      [ "Чаппелл", "Chappell" ],
+      [ "Чемберс", "Chambers" ],
+      [ "Чен", "Chen" ],
+      [ "Ченг", "Cheng" ],
```

Comment:
I believe both of these would be transcribed as [`Чэн`](https://ru.wikipedia.org/wiki/%D0%A7%D1%8D%D0%BD_(%D1%84%D0%B0%D0%BC%D0%B8%D0%BB%D0%B8%D1%8F)). (Then again, my romanization of Chinese is basically nonexistent, so any additional insight is appreciated.)

Reply:
These two are pronounced differently in Chinese though. The former ends with an `n` sound like the one in `kin` and the latter ends with an `ng` sound like the one in `king`. Strictly speaking the vowels are also different (the former's being `/ə/` whereas the latter's being `/ɤ/`). Wiktionary suggests [`чэнь` for `Chen`](https://en.wiktionary.org/wiki/%E9%99%B3#Chinese) and [`чэн` for `Cheng`](https://en.wiktionary.org/wiki/%E7%A8%8B) (You can click the `more` button in the `Pronunciation` section to see the transcription of the Mandarin pronunciation). It is not consistent with how English names ending with `n` or `ng` are transcribed in the list, but I think I'll go with Wiktionary's version since it uses the official [Palladius system](https://en.wikipedia.org/wiki/Cyrillization_of_Chinese) of Chinese-to-Russian transcription.